packages/common/src/projects/fetch.ts

```diff
@@ -55,6 +55,19 @@ export async function convertItemToProject(
   requestOptions: IRequestOptions
 ): Promise<IHubProject> {
   const model = await fetchModelFromItem(item, requestOptions);
+  // Set up obj tracking resource prop && filename (KEEP IN SYNC WITH getPropertyMap()
+  // for additional resources!)
+  const resourceNameAndPropPairs: {
+    [key: string]: string;
+  } = {
+    location: "location.json",
+  };
```

Comment:
Per @dbouwman we want to be very explicit about prop/filename matching. We should consider a centralized location to keep this obj, the one in edit in sync along with the prop names in getBasePropertyMap.